lib/kino_db/sql_cell.ex

```diff
@@ -149,11 +156,34 @@ defmodule KinoDB.SQLCell do
     to_quoted(attrs, quote(do: Exqlite), fn n -> "?#{n}" end)
   end
 
+  defp to_quoted(%{"connection" => %{"type" => "bigquery"}} = attrs) do
+    to_quoted(attrs, quote(do: ReqBigQuery), fn _n -> "?" end)
+  end
+
   defp to_quoted(_ctx) do
     quote do
     end
   end
 
+  defp to_quoted(attrs, quote(do: ReqBigQuery), next) do
+    {query, params} = parameterize(attrs["query"], next)
+    bigquery = {quoted_query(query), params}
+    opts = query_opts_args(attrs)
+
+    req_opts =
+      if opts == [],
+        do: [[bigquery: bigquery]],
+        else: [Keyword.put(Enum.at(opts, 0), :bigquery, bigquery)]
+
+    quote do
+      unquote(quoted_var(attrs["result_variable"])) =
+        Req.post!(
+          unquote(quoted_var(attrs["connection"]["variable"])),
+          unquote_splicing(req_opts)
```

Comment:
`unquote_splicing` is only useful if you want to make a function call with multiple arguments. Here, opts is just a single argument. A regular `unquote` is enough.